.gitignore

```diff
@@ -50,7 +50,7 @@ luac.out
 /robase-venv
 
 # JUnit test report
-/testReport.xml
```

Comment:
?

Reply:
The output file for test runs has been renamed in the Jenkinsfile.